cumulusci/salesforce_api/package_zip.py

```diff
@@ -87,17 +87,20 @@ def _populate_zip(self):
 
 
 class InstallPackageZipBuilder(BasePackageZipBuilder):
-    api_version = "43.0"
-
-    def __init__(self, namespace, version, activateRSS=False, password=None):
+    def __init__(
+        self, namespace, version, api_version, activateRSS=False, password=None
+    ):
```

Comment:
I did a search in both our github orgs to see if there are any other repos using this class, and didn't find any, so I think this change in the arguments should be okay as long as we've updated all the call sites in CumulusCI itself.